testdata/go/v2/memcached-operator/PROJECT

```diff
@@ -1,11 +1,17 @@
 domain: example.com
 layout: go.kubebuilder.io/v2
+plugins:
+  go.sdk.operatorframework.io/v2-alpha: {}
 projectName: memcached-operator
 repo: github.com/example/memcached-operator
 resources:
-- group: cache
+- api:
+    crdVersion: v1beta1
+    namespaced: true
+  controller: true
+  domain: example.com
+  group: cache
   kind: Memcached
+  path: github.com/example/memcached-operator/api/v1alpha1
   version: v1alpha1
-version: 3-alpha
-plugins:
-  go.sdk.operatorframework.io/v2-alpha: {}
+version: "3"
```

Comment:
It needs to be checked. The v2 should not have new fields.

Reply:
Discussed offline: the fields in the project configuration file are defined by the config version (in this case v3) not by the plugin version (in this case v2).